subprocess/interface_test.go

```diff
@@ -66,3 +68,30 @@ func TestOutputOptions(t *testing.T) {
 	opts = OutputOptions{SuppressError: true, SuppressOutput: true}
 	assert.NoError(opts.Validate())
 }
+
+func TestOutputOptionsIntegrationTableTest(t *testing.T) {
+	buf := &bytes.Buffer{}
+	shouldFail := []OutputOptions{
+		{Output: buf, Error: buf},
+		{Output: buf, SendErrorToOutput: true},
+	}
+
+	shouldPass := []OutputOptions{
+		{SuppressError: true, SuppressOutput: true},
+		{Output: buf, SendErrorToOutput: true},
+		{Output: &util.CappedWriter{Buffer: buf, MaxBytes: 1024 * 1024}, SendErrorToOutput: true},
+	}
+
+	assert := assert.New(t) // nolint
+
+	for _, opt := range shouldFail {
+		assert.Error(opt.Validate(), "%+v", opt)
```

Comment:
This test fails when I run it

--- FAIL: TestOutputOptionsIntegrationTableTest (0.00s)
        Error Trace:    interface_test.go:88
	Error:      	An error is expected but got nil.
	Messages:   	{Output: Error:<nil> SuppressOutput:false SuppressError:false SendOutputToError:false SendErrorToOutput:true}